### PR TITLE
mac_hdflop.xml: attach the list and start adding stuff

### PR DESCRIPTION
**Pull request by rzero9**
This change attaches the orphan mac_hdflop.xml software list to (hopefully) all the Macintosh machines with HD floppy drives, and adds about 70 new software items to it.

The Mac is a bit out of my comfort zone, but rockleevk sent me some previously undumped KanjiTalk images a few months ago, and now that the Mac drivers fully support HD disks and all the obvious bugs have been ironed out, I thought this was a good time to add them along with some other things I got from WinWorld. I actually have about 60 more more waiting to be added, but as this is an almost-new list I wanted to submit my work so far and see if everything checks out.

A couple of notes about the naming...

- I have tried to reproduce disk labels as part_ids as closely as possible. In some cases this involves languages that are completely unknown to me, so I might have made some mistakes there. At the very least I know that the English, Spanish and Japanese labels shoud be 100% correct. 

- Application names can be a bit inconsistent, so I have tried to go with what the installer or README files call them (including the company name if it's there, like "Microsoft PowerPoint" or "Adobe Illustrator").

- About the name of the operating system, I think Apple only started calling it "Mac OS" officially with version 7.6, so I have followed that convention. Also, the Korean and Japanese versions have their own regional names (HangulTalk and KanjiTalk, respectively), so I follow those too.

And as a side note, I know mac_flop.xml needs a lot of love too. I may start working on that when I finish sorting out the HD stuff.

### CODE REVIEW

File: hash/mac_hdflop.xml [hunk 1]
```diff
@@ -5,125 +5,3347 @@ license:CC0
 -->
 <softwarelist name="mac_hdflop" description="Macintosh High Density Disk images">
 
+<!-- Operating system disks -->
+
+	<software name="htalk71">
+		<description>HangulTalk 7.1</description>
```
Comment (cuavas) [inline hash/mac_hdflop.xml:11]: Is this officially called “HangulTalk” in Latin script?  If not, please use RR transliteration “Hangeul”.

File: hash/mac_hdflop.xml [hunk 2]
```diff
@@ -5,125 +5,3347 @@ license:CC0
...
+		<part name="flop6" interface="floppy_3_5">
+			<feature name="part_id" value="Utilities" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Utilities.img" size="1474560" crc="3cadd68a" sha1="1ecef7d3a19cf44dc8942faeb70c2653aee3ee2b" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="sys712lcq" supported="no">
+		<description>System Software 7.1.2 (LC 580/Quadra 630, US English)</description>
```
Comment (cuavas) [inline hash/mac_hdflop.xml:2022]: Was this ever distributed in this form, or are these generated from the Disk Copy images for creating a set of installation floppies on the CD?  Did the LC580 actually come in a configuration without a CD-ROM drive?  I only ever saw the system software on CD, with Disk Copy images for creating a set of floppies if you needed them for some reason.

File: hash/mac_hdflop.xml [hunk 3]
```diff
@@ -5,125 +5,3347 @@ license:CC0
...
+			<feature name="part_id" value="Disk 6" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk06.img" size="1474560" crc="a83aed12" sha1="c1991ee1f6712ea0b6372dd423e71a82c89f509e" />
+			</dataarea>
+		</part>
+	</software>
+
 	<!-- S/N: Z93350-001A -->
-	<software name="clarisw" supported="no">
+	<software name="clarisw">
```
Comment (cuavas) [inline hash/mac_hdflop.xml:2641]: Can you put the version and language in the name?  ClarisWorks 1 and 2 are very different pieces of software.

File: hash/mac_hdflop.xml [hunk 4]
```diff
@@ -5,125 +5,3347 @@ license:CC0
...
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2" />
+			<dataarea name="flop" size="1474560">
+				<rom name="disk02.img" size="1474560" crc="3faeb185" sha1="bc9bc42e1835cfbf7390e3dd5a95083e888c704e" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="macwp15v3">
```
Comment (cuavas) [inline hash/mac_hdflop.xml:2902]: Should the different revisions of 1.5 be clones of the latest one?

File: hash/mac_hdflop.xml (hunk 1 above)
Comment (rzero9) [reply hash/mac_hdflop.xml:11]: Yes, it's officially written like that in Latin script. See: https://imgur.com/1BeL2Cb

File: hash/mac_hdflop.xml (hunk 2 above)
Comment (rzero9) [reply hash/mac_hdflop.xml:2022]: I'm honestly not sure about this, but it's definitely possible. I could add a note indicating that those images are probably extracted from the CD.

File: hash/mac_hdflop.xml (hunk 3 above)
Comment (rzero9) [reply hash/mac_hdflop.xml]: This was one of the few entries that were already there before, so I inadvertently skipped it. It should be consistent with the rest now.

File: hash/mac_hdflop.xml (hunk 4 above)
Comment (rzero9) [reply hash/mac_hdflop.xml]: Makes sense, modified it so macwp15v1 is a clone now.

**Pull request merged**